building_dialouge_webapp/heat/navigation.py

```diff
@@ -0,0 +1,29 @@
+def get_flows():
+    from . import flows
+    from . import views
+
+    return [
```

Comment:
I would recommend following structure in order to distinguish between category (view instead of flow) and flow:
```
[
    {
        "category": "Verbrauch",
        "url": "<url>", 
        "steps": [
            {"name": "Keller", "flow": CellarFlow, "url": "<url>"},
            ...
        ]
    },
    {"category": ...}
]
```